helper-cli/src/main/kotlin/HelperMain.kt

```diff
@@ -33,6 +33,7 @@ import org.apache.logging.log4j.core.config.Configurator
 import org.ossreviewtoolkit.helper.commands.ExportLicenseFindingCurationsCommand
 import org.ossreviewtoolkit.helper.commands.ExportPathExcludesCommand
 import org.ossreviewtoolkit.helper.commands.ExtractRepositoryConfigurationCommand
+import org.ossreviewtoolkit.helper.commands.ImportScanResultsCommand
```

Comment:
Commit message:
* "This import command"
* What do you mean by indexed?

Reply:
With indexed I mean the common sense one, e.g. the existence of a data structure which allows for efficient access based on some given key.

Reply:
Ok, I expected some additional indexing when I read this.